less/home-page.less

```diff
@@ -19,17 +23,40 @@
     flex-wrap: wrap;
     width: 100%;
 
+    .blast {
+        @media @phone{
+            visibility: visible;
+            width: 90%;
+            margin: 4% auto;
+        }
+        
+    }
+
+    .sky, .launch {
+        @media @phone {
```

Comment:
Good job targeting the two images by their class names. &:not(:first-child){display: none;} would have also worked here.